common/rootfs/docker-entrypoint.sh

```diff
@@ -70,6 +70,12 @@ if bashio::config.true 'disable_tuya_default_response'; then
     export DISABLE_TUYA_DEFAULT_RESPONSE="true"
 fi
 
+TIMEZONE=$(bashio::config 'timezone')
```

Comment:
It seems it also possible to retrieve the timezone from the supervisor: https://github.com/hassio-addons/bashio/blob/main/lib/supervisor.sh#L171. 

I think this would be better since there would be no need to manually configure it anymore.

Reply:
Thanks, I have updated it.